kolibri/core/auth/api.py

```diff
@@ -404,6 +406,42 @@ def annotate_queryset(self, queryset):
         )
 
 
+class IndividualLearnersGroupViewSet(viewsets.ModelViewSet):
+    permission_classes = (KolibriAuthPermissions,)
+    filter_backends = (KolibriAuthPermissionsFilter, DjangoFilterBackend)
+    queryset = IndividualLearnersGroup.objects.all()
+    serializer_class = IndividualLearnersGroupSerializer
+
+    filter_fields = ("parent",)
+
+    values = ("id", "name", "parent", "user_ids")
+
+    field_map = {"user_ids": partial(process_uuid_aggregate, key="user_ids")}
+
+    def annotate_queryset(self, queryset):
```

Comment:
@rtibbles I've included this because it appeared in the LearnerGroupViewSet while I was rebasing - please advise if this is unnecessary here or if there is anything that needs to be changed.

Note that the InvididualLearnersGroup is also a child of Collection - so it seems that this code ought to work as expected assuming it is expected to be here in the first place.